application/migrations/20210712231000_add_fields_to_locations.up.fizz

```diff
@@ -0,0 +1,5 @@
+add_column("locations", "state", "string")
```

Comment:
What did you think of making the lat/long not nullable? Better done in a separate PR?

Reply:
Yeah, separate PR.